nipype/pipeline/engine/nodes.py

```diff
@@ -1125,6 +1125,8 @@ def _make_nodes(self, cwd=None):
                     fieldvals = filename_to_list(getattr(self.inputs, field))
                 logger.debug('setting input %d %s %s', i, field, fieldvals[i])
                 setattr(node.inputs, field, fieldvals[i])
+            node._interface.num_threads = self._interface.num_threads
```

Comment:
Oh, I see the problem: In `Node.__init__`, `self._interface.num_threads = n_procs`, which is 1 by default...